addons/fluxcd/resources/components/helm-controller.cue

```diff
@@ -0,0 +1,68 @@
+package main
+
+helmController: {
+	name: "helm-controller"
+	type: "webservice"
+	dependsOn: ["fluxcd-ns"]
+	properties: {
+		imagePullPolicy: "IfNotPresent"
+		image:           parameter.registry + "/fluxcd/helm-controller:v0.22.0"
```

Comment:
```suggestion
		image:           parameter.registry + "fluxcd/helm-controller:v0.22.0"
```

images from docker hub don't have `/` prefix

Reply:
the same to others

Reply:
implemented logic to determinate if the "/" is required